.gitlab-ci/molecule.yml

```diff
@@ -0,0 +1,22 @@
+---
+molecule_docker:
+  stage: unit-tests
+  # TODO: This should be replaced with a community built image
+  # Dockerfile for this image can be found here:
+  # https://gitlab.esss.lu.se/ics-docker/kubespray-molecule/blob/master/Dockerfile
+  image: registry.esss.lu.se/ics-docker/kubespray-molecule:latest
```

Comment:
I can arrange hosting of that in `quay.io/kubespray`. Is there a Dockerfile available?

Reply:
The Dockerfile is available here:
https://gitlab.esss.lu.se/ics-docker/kubespray-molecule/blob/master/Dockerfile